### ISSUE TITLE
User Not Authorised to refresh CSRF Tokens

### ISSUE THREAD

**Issue by jhealey5** [verbb/formie#624]
**Description**
Refreshing the CSRF/Capcha tokens with the updated controller as described here https://verbb.io/craft-plugins/formie/docs/template-guides/cached-forms#csrf-token can only be done with a logged in user. When not logged in you get an "Unauthorised. User is not permitted to perform this action" 403 error.

**Steps to reproduce**
1. Follow the steps here to manually refresh tokens on a cached page https://verbb.io/craft-plugins/formie/docs/template-guides/cached-forms#csrf-token

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.23
- Craft version: Craft Pro 3.7.18.2
- Multi-site: No

**Additional context**

Whilst the `FormsController.php` does allow anonymouse access to the refresh-tokens endpoint, it's being later restricted by the `init()` method I think. I don't know what adverse effects this has but removing `$this->requirePermission('formie-viewForms');` on line 43 seems to solve the problem.

**Comment by engram-design** [comment]
Sorry about that, fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.23",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.24](https://github.com/verbb/formie/releases/tag/1.4.24)
